walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -62,9 +59,9 @@
         {
```

Comment:
Since `newEvents.get(0).getWhen()` is used multiple times, can we create a var for it?